docs/content/en/_index.html

```diff
@@ -4,150 +4,826 @@
+                                 aria-hidden="true">
+                                <path d="M1 5.12939C0.585787 5.12939 0.25 5.46518 0.25 5.87939C0.25 6.29361 0.585787 6.62939 1 6.62939V5.12939ZM12 5.12939L1 5.12939V6.62939L12 6.62939V5.12939Z"
+                                      fill="currentColor"/>
+                                <path d="M7.11108 1L12 5.7735L7.11108 10.7444" stroke="currentColor" stroke-width="1.5"
+                                      stroke-linecap="round" stroke-linejoin="round"/>
+                            </svg>
+                        </span>
+                    </div>
+                </a>
+            </li>
+        </ul>
+    </div>
+
+    <div class="tabs">
+        <h2 class="title tabs__title">How to Install Tetragon</h2>
+        <ul class="tabs__list">
+            <li class="tabs__button-list">
+                <button class="tabs__button tabs__button--active" type="button">
+                    <span class="tabs__tab-title">Install Tetragon on Kubernetes</span>
+                    <span class="tabs__tab-text">Download via helm chart</span>
+                    <span class="tabs__button-mobile" aria-hidden="true">Kubernetes</span>
+                </button>
+                <button class="tabs__button" type="button">
+                    <span class="tabs__tab-title">Install on any Linux system</span>
+                    <span class="tabs__tab-text">Install and manage Tetragon via released packages</span>
+                    <span class="tabs__button-mobile" aria-hidden="true">Linux</span>
+                </button>
+                <button class="tabs__button" type="button">
+                    <span class="tabs__tab-title">Deploy as a container</span>
+                    <span class="tabs__tab-text">Run the Tetragon image from the Tetragon quay repository</span>
+                    <span class="tabs__button-mobile" aria-hidden="true">Copy container</span>
+                </button>
+            </li>
+            <li class="tabs__content-list">
+                <div class="tabs__content">
+                    <div class="tabs__content-mobile">
+                        <span class="tabs__tab-title">Install Tetragon on Kubernetes</span>
+                        <span class="tabs__tab-text">Download via helm chart</span>
+                    </div>
+                    {{< tab-code header="Kubernetes" lang="shell" >}}
+                    helm repo add cilium https://helm.cilium.io
+                    helm repo update
+                    helm install tetragon cilium/tetragon -n kube-system
+                    kubectl rollout status -n kube-system ds/tetragon -w
+                    {{< /tab-code >}}
+                </div>
+                <div class="tabs__content" style="display: none">
+                    <div class="tabs__content-mobile">
+                        <span class="tabs__tab-title">Install on any Linux OS system </span>
```

Comment:
you could have used CTRL+F here
```suggestion
                        <span class="tabs__tab-title">Install on any Linux system </span>
```